### PR TITLE
Resolves #1322 : Added more details in organisations/publishers page …

### LINKED ISSUE #1322: Organisation page
### Problem description
Currently, the organisation page is missing some important information. 

What the page will include (not in order):

Title
Dataset count
Description
Link to datasets
Email
Phone number
Website

*For mobile:* Omit the 'Ask a question' button that's available in desktop.

### Screenshot / Design / File reference
https://www.figma.com/file/6V1RKGaGFsE5IJiPG3Tc4u/Organisation

mobile
desktop 1

### PR DESCRIPTION
**Pull request by nahidakbar**
…to reflect design

### Checklist

-   [x] There are unit tests to verify my changes are correct | Unit tests aren't applicable (delete one)
-   [x] I've updated CHANGES.md with what I changed.
-   [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column
-   [x] There are sufficient comments for my code to be understandable - and I realise reviewers will pull me up on it if not!

### CODE REVIEW

File: magda-web-client/src/Components/Publisher/PublisherDetails.js [hunk 1]
```diff
@@ -60,9 +75,34 @@ class PublisherDetails extends Component {
...
+                        {(details.email ||
+                            details.website ||
+                            details.phone) && (
+                            <div className="publisher-details-contacts">
+                                <h3 className="section-heading">
+                                    Contact details
+                                </h3>
+                                {details.email && (
+                                    <div className="publisher-details-contacts-item">
+                                        Email: {details.email}
```
Comment (chloeleichen) [inline magda-web-client/src/Components/Publisher/PublisherDetails.js]: We should make it a mailto link: https://www.w3schools.com/tags/tag_address.asp

File: magda-web-client/src/Components/Publisher/PublisherDetails.js [hunk 2]
```diff
@@ -108,10 +150,16 @@ function mapStateToProps(state: Object, ownProps: Object) {
     const isFetching: boolean = state.publisher.isFetchingPublisher;
     const error: object = state.publisher.errorFetchingPublisher;
     const location: Location = ownProps.location;
+    const datasetSearch: Object = state.datasetSearch;
+    const searchPublisherName =
+        datasetSearch.queryObject && datasetSearch.queryObject.publisher;
+    const hits = datasetSearch.hitCount;
```
Comment (chloeleichen) [inline magda-web-client/src/Components/Publisher/PublisherDetails.js]: Can we call it hitCount since it's called hitCount everywhere else?

**Comment by nahidakbar** [comment]
This is ready to go!
@AlexGilleran @chloeleichen

**Comment by AlexGilleran** [comment]
Merging this because @chloeleichen's at uni til Wednesday - nice work @nahidakbar !

**Pull request merged**